core/kazoo_ast/src/cb_api_endpoints.erl

```diff
@@ -31,12 +33,18 @@
 -spec to_ref_doc() -> 'ok'.
 -spec to_ref_doc(atom()) -> 'ok'.
 to_ref_doc() ->
+    ensure_ref_dir(),
     lists:foreach(fun api_to_ref_doc/1, ?MODULE:get()).
 
 to_ref_doc(CBModule) ->
+    ensure_ref_dir(),
     Path = code:which(CBModule),
     api_to_ref_doc(hd(process_module(Path, []))).
 
+-spec ensure_ref_dir() -> 'ok'.
+ensure_ref_dir() ->
+    'ok' = filelib:ensure_dir(filename:join([?REF_PATH, ".placeholder"])).
```

Comment:
When using `ensure_dir` it will create the parent directory only - if the path ends with a directory - so `ensure_dir(<<"/a/b/c">>)` will create `/a/b`. This is just a sanity check to make sure the ref path exists (could be someone built a release and wants docs for some reason?). Doesn't hurt to have it.